headers/functions/arm9.h

```diff
@@ -108,6 +108,8 @@ void OpenPackFile(struct pack_file_opened* pack_file, const char* file_name);
 uint32_t GetFileLengthInPack(struct pack_file_opened* pack_file, uint32_t file_index);
 uint32_t LoadFileInPack(struct pack_file_opened* pack_file, void* output_buffer,
                         uint32_t file_index);
+void GetDungeonResultMsg(union damage_source damage_source_or_result, void* buffer, int buffer_size,
```

Comment:
Is `buffer` always a string output? If so I think using `char*` would probably make more sense.

Reply:
Right